draft-ietf-tls-esni.md

```diff
@@ -787,6 +815,48 @@ where HKDF-Expand-Label and HKDF-Extract are as defined in {{RFC8446}}. The
 value of ServerHello.random[0:24] is generated as usual by invoking a secure
 random number generator (see {{RFC8446}}, Section 4.1.2).
 
+When confirming ECH acceptance, the backend server SHOULD reply to the
+"handshake_padding" extension solicited by the client, computing the padding
+length as described in {{server-padding}}.
+
+### Recommended Padding Scheme {#server-padding}
+
+It is RECOMMENDED that the backend server compute the padding length for the
+"handshake_padding" extension ({{handshake-padding}}) based on the following
+values:
```

Comment:
This should probably be clear this is going in EncryptedExtensions, not Certificate, as they both carry padding.